manifests/password.pp

```diff
@@ -40,18 +40,18 @@
   }
 
   file { "$password_config_file":
-    content => $password_content,
     ensure  => present,
+    content => $password_content,
     require => Package[$package_name],
     notify  => Service[$virtual_service],
     tag     => "splunk_password",
   }
 
   file { "$secret_file":
+    ensure  => present,
```

Comment:
I guess we don't use https://github.com/voxpupuli/puppet-lint-file_ensure-check?